Intersect.Client/Entities/Entity.cs

```diff
@@ -846,7 +846,7 @@ public virtual void Draw()
                 //If unit is stealthed, don't render unless the entity is the player.
                 if (Status[n].Type == StatusTypes.Stealth)
                 {
-                    if (this != Globals.Me && !Globals.Me.IsInMyParty(this))
+                    if (this != Globals.Me || this is Player player && !Globals.Me.IsInMyParty(player))
```

Comment:
This was my bad... I think this should have actually been this?

`if (this != Globals.Me && !(this is Player player && Globals.Me.IsInMyParty(player)))`